src/utils/middleware/__tests__/setSentryUser.test.js

```diff
@@ -0,0 +1,55 @@
+import * as Sentry from '@sentry/node'
+import getMockReq from 'src/utils/testHelpers/mockReq'
+import getMockRes from 'src/utils/testHelpers/mockRes'
+import {
+  CUSTOM_REQ_DATA_KEY,
+  AUTH_USER_KEY,
+} from 'src/utils/middleware/constants'
+
+jest.mock('src/utils/logger')
+jest.mock('next-firebase-auth', () => ({
```

Comment:
This is totally fine, but just as an educational FYI: we have custom mocks set up for `next-firebase-auth`, so you could exclude the custom mocking here and import `useAuthUser` instead, if you want.